mbed_app.json

```diff
@@ -1,5 +1,29 @@
 {
 	"config": {
+        "lora-radio": {
+            "help": "Which radio to use (options: SX1272,SX1276)",
+            "value": "SX1272"
+        },
+
+        "lora-spi-mosi":       { "value": "D11" },
```

Comment:
These are the configs for sx1276 radio shield, i think they're the same for sx1272. So now if you have the shield just change the setting above and done.